.cirrus.yml

```diff
@@ -24,21 +20,16 @@ env:
     # Runner statistics log file path/name
     STATS_LOGFILE_SFX: 'runner_stats.log'
     STATS_LOGFILE: '$GOSRC/${CIRRUS_TASK_NAME}-${STATS_LOGFILE_SFX}'
-    # Netavark/aardvark location/options when TEST_ENVIRON=host-netavark
-    NETAVARK_URL: "https://api.cirrus-ci.com/v1/artifact/github/containers/netavark/success/binary.zip?branch=${NETAVARK_BRANCH}"
-    NETAVARK_DEBUG: 0  # set non-zero to use the debug-mode binary
-    AARDVARK_URL: "https://api.cirrus-ci.com/v1/artifact/github/containers/aardvark-dns/success/binary.zip?branch=${AARDVARK_BRANCH}"
-    AARDVARK_DEBUG: 0  # set non-zero to use the debug-mode binary
 
     ####
     #### Cache-image names to test with (double-quotes around names are critical)
     ####
-    FEDORA_NAME: "fedora-35"
-    PRIOR_FEDORA_NAME: "fedora-34"
+    FEDORA_NAME: "fedora-36"
+    PRIOR_FEDORA_NAME: "fedora-35"
```

Comment:
Not for this PR, but it would be nice to be able to figure out programmatically what these values should be rather than having to hard code them.  That said, I don't have a good suggestion on how to pull that off.

Reply:
This file is the source of definitions for automation, so it really can't be done using imperial logic.  Cirrus-CI does have support for using a scripting-language instead of YAML, but looking up the values would be complex at best, and error-prone at worst.  So hard-coding the values is the least-worst solution that makes the most sense at the moment.